doc/lightningd-config.5.md

```diff
@@ -212,6 +212,9 @@ If there is no `hsm_secret` yet, `lightningd` will create a new encrypted secret
 If you have an unencrypted `hsm_secret` you want to encrypt on-disk, or vice versa,
 see lightning-hsmtool(8).
 
+ **max-dust-htlc-exposure-msat**=*MILLISATOSHI*
+Option which limits the total amount of sats to be allowed as dust on a channel.
+
```

Comment:
Doesn't this really belong in the **Lightning channel and HTLC options** section, perhaps adjacent to **`max-concurrent-htlcs`**?

Reply:
Mh, I agree with that thanks